src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx

```diff
@@ -488,6 +524,8 @@ const MissingItemFormEdit = () => {
                       name="description"
                       value={formData.description}
                       onChange={(e) => setFormData({ ...formData, description: e.target.value })}
+                      error={!!validationErrors.description}
```

Comment:
A more readable way to get a boolean from a non-boolean value is with the Boolean constructor:
```suggestion
                      error={Boolean(validationErrors.description)}
```

(Note: the Boolean constructor without the `new` keyword returns the value coerced to a primitive `boolean` type in JS. If you use the `new` keyword, you get a non-primitive `Boolean` object.)

Reply:
If it's just in the lost-and-found code, you can add it the backlog of changes to make for the next release. If it's more widespread throughout 360, I would say don't worry about it.